std/src/main/scala/algebra/std/byte.scala

```diff
@@ -0,0 +1,58 @@
+package algebra
+package std
+
+import algebra.lattice._
+import algebra.number._
+import algebra.ring._
+import algebra.std.util.StaticMethods
+
+package object byte extends ByteInstances
+
+trait ByteInstances {
+  implicit val byteAlgebra = new ByteAlgebra
+
+  val ByteMinMaxLattice = new Lattice[Byte] {
```

Comment:
what's the thought on anonymous instances vs classes? I prefer anonymous trait style personally, but there may be scala-jvm implementation details that makes the class approach better. I'd like a rule that we know when to choose each.